robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -14,26 +14,32 @@
 import edu.wpi.first.math.controller.PIDController;
 import edu.wpi.first.units.AngleUnit;
 import edu.wpi.first.units.Units;
+import edu.wpi.first.units.measure.Current;
 import edu.wpi.first.units.measure.Distance;
 
 /**
- * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean
- * constants. This class should not be used for any other purpose. All constants should be declared
+ * The Constants class provides a convenient place for teams to hold robot-wide
+ * numerical or boolean
+ * constants. This class should not be used for any other purpose. All constants
+ * should be declared
  * globally (i.e. public static). Do not put anything functional in this class.
  *
- * <p>It is advised to statically import this class (or one of its inner classes) wherever the
+ * <p>
+ * It is advised to statically import this class (or one of its inner classes)
+ * wherever the
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
 
   public static class CAN_ID {
-    public static final int ELEVATOR_LEFT_MOTOR = 23;
-    public static final int ELEVATOR_RIGHT_MOTOR = 24;
-    public static final int ALGAE_RUNNER_MOTOR = 25;
+    public static final int ELEVATOR_LEFT_MOTOR = -1;
+    public static final int ELEVATOR_RIGHT_MOTOR = -1;
+    public static final int ALGAE_RUNNER_MOTOR = -1;
+    public static final int LEFT_ALGAE_PIVOT_MOTOR = -1;
+    public static final int RIGHT_ALGAE_PIVOT_MOTOR = -1;
   }
 
   public static class OPERATOR_CONSTANTS {
-
     public static final int kDriverControllerPort = 0;
   }
```

Comment:
We can just get rid of this class. We'll end up creating our own for controller constants.